src/GT.UI/Program.cs

```diff
@@ -73,8 +136,8 @@
 		pattern: "/{action}", // URL with parameters
 		defaults: new { controller = "Home", action = "Index" }); // Parameter defaults
```

Comment:
Isn't this leftover code? Should be removed?

Reply:
We have two ways of resolving URLs in order to meet requirements on app.
- first pointing to /Action (applied to our default, allowing localhost/ as index and localhost/kontakt as contact.
- second one addressing /API/Controller/Action/Id? (normally default) - this could be removed later in case we don't want to traverse over /api/ url.

Reply:
Not per default URL-scheme and not by "id?", which is why I suggest we keep this.

This is also not part of #57, but from #25. It may look as it's from this commit due to Line endings being resolved.
So, in case we edit this - it should be done related to that or a new issue.